packages/swap/src/routes/v2/swap.ts

```diff
@@ -310,8 +315,8 @@ router.get(
             }),
         fees: aggregateFees,
       },
-      ...(showSwapEgress && { swapEgress: { ...swapEgressFields } }),
-      ...(showRefundEgress && { refundEgress: { ...refundEgressFields } }),
+      ...(swapEgressFields && { swapEgress: { ...swapEgressFields } }),
+      ...(refundEgressFields && { refundEgress: { ...refundEgressFields } }),
```

Comment:
i think you could have empty objects now.

Reply:
We shouldn't, I changed the function to return null if none of the data is present